.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,79 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+# It will also deploy the build to two different repos with https://github.com/peaceiris/actions-gh-pages
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [13.3.0]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm ci
+        working-directory: ./web
+      - run: npm run check-format
+        working-directory: ./web
+      - run: npm run lint
+        working-directory: ./web
+      - run: npm run test
+        working-directory: ./web
+      - run: npm run build
+        working-directory: ./web
+    env:
+      NODE_OPTIONS: '--max-old-space-size=4096'
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    # Uncomment this before merging.
+    #if: github.ref == 'refs/heads/master'
+    steps:
+      - name: Prepare imjoy.lib deploy
+        run: |
+          # Copy dirs and files and that we want to update in imjoy.lib.
+          mkdir imjoy-core && \
+          cp -Rf ./web/dist/* ./imjoy-core && \
```

Comment:
We have the following error:
```
Run # Copy dirs and files and that we want to update in imjoy.lib.
cp: cannot stat './web/dist/*': No such file or directory
##[error]Process completed with exit code 1.
```

And I found our previous deploy-core.sh:
```cp -Rf ../web/dist/*  ./```
Is that the issue?

Reply:
We're in the root of the project here. Before we were inside a subdir where we cloned the repo.

Reply:
It could be that the build result isn't saved from the previous job. I'll check the docs on that.

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts